go/test/endtoend/vreplication/vreplication_test.go

```diff
@@ -65,6 +65,7 @@ const (
 
 	merchantKeyspace = "merchant-type"
 	maxWait          = 10 * time.Second
```

Comment:
We may just need to increase the max wait time. 10 seconds is pretty low given how flaky GitHub actions can be. 🙂